src/rebar_packages.erl

```diff
@@ -234,7 +234,7 @@ parse_checksum(Checksum) ->
 update_package(Name, RepoConfig=#{name := Repo}, State) ->
     ?MODULE:verify_table(State),
     try hex_repo:get_package(get_package_repo_config(RepoConfig), Name) of
-        {ok, {200, _Headers, Releases}} ->
+        {ok, {200, _Headers, #{releases := Releases}}} ->
```

Comment:
I believe this is correct, the following works on hex_core v0.4.0:

```erlang
> {ok, {_, _, Releases}} = hex_repo:get_package(hex_core:default_config(), <<"hex_core">>), [Version || #{version := Version} <- Releases].
[<<"0.1.0">>,<<"0.1.1">>,<<"0.2.0">>,<<"0.2.1">>,
 <<"0.3.0">>,<<"0.4.0">>]
````

we've broken API between 0.3.0 and 0.4.0, so maybe that caused confusion? We'll get better at backwards compatibility going forward.

Reply:
@wojtekmach yea, having issues bootstrapping. With this change it then breaks in a different spot. Not sure how this is the case yet, there should only be 0.4.0.